apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -1275,12 +1275,17 @@ expression: "&schema"
           "properties": {
             "display_filename": {
               "description": "Display the filename in the logs",
-              "default": true,
+              "default": false,
               "type": "boolean"
             },
             "display_line_number": {
               "description": "Display the line number in the logs",
-              "default": true,
+              "default": false,
+              "type": "boolean"
+            },
+            "display_target": {
+              "description": "Display the target in the logs",
+              "default": false,
```

Comment:
Configuration changes make me nervous. Would this break any existing configurations? Is it possible and do we need to write a migration for it? (Likewise for the change below)

Reply:
Previously this would take effect only if the logging section is present in the config.
The new behavior is more consistent.

Before:
```
telemetry:
  experimental_logging:
    display_line_number: true
```
Made `display_filename` true

Now:
```
telemetry:
  experimental_logging:
    display_line_number: true
```
Only makes `display_line_number` true.

As this is an experimental section I think we are OK to fix this.

Reply:
I think that's fine. It just needs to be documented in the changeset.